src/core/Table/Table.tsx

```diff
@@ -648,30 +656,59 @@ export const Table = <
             }
           }}
         >
-          {data.length !== 0 &&
-            page.map((row: Row<T>) => {
-              prepareRow(row);
-              return (
-                <TableRowMemoized
-                  row={row}
-                  rowProps={rowProps}
-                  isLast={row.index === data.length - 1}
-                  onRowInViewport={onRowInViewportRef}
-                  onBottomReached={onBottomReachedRef}
-                  intersectionMargin={intersectionMargin}
-                  state={state}
-                  key={row.getRowProps().key}
-                  onClick={onRowClickHandler}
-                  subComponent={subComponent}
-                  isDisabled={!!isRowDisabled?.(row.original)}
-                  tableHasSubRows={hasAnySubRows}
-                  tableInstance={instance}
-                  expanderCell={expanderCell}
-                />
-              );
-            })}
+          {data.length !== 0 && (
+            <>
+              {useVirtualization && (
+                <VirtualScroll>
+                  {page.map((row: Row<T>) => {
+                    prepareRow(row);
+                    return (
+                      <TableRowMemoized
```

Comment:
Move row jsx to a variable or smth to avoid duplication.

Reply:
Changed, but..looks bad :(